static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -463,7 +467,9 @@ export const getEpicTestsFromGoogleDoc = (): Promise<
                         audienceCriteria: 'All',
                         audience: 1,
                         audienceOffset: 0,
-
+                        useLocalViewLog: rows.some(row =>
```

Comment:
I realised this needs to be above the `isThankYou` segment since otherwise it will overwrite its `useLocalViewLog` value